docs/essentials/index.md

```diff
@@ -0,0 +1,34 @@
+---
+id: essentials_index
+title:  "Overview"
+---
+
+ZIO-NIO is a small, unopinionated ZIO interface to NIO. 
+
+ - **[File Channel](files.md)** — For processing files that are available locally. For every operation a new fiber is started to perform operation
+ - **[Socket Channel](sockets.md)** — Provides API for remote communication with `InetSocket`s 
+ 
+## So what is NIO once again? 
+
+As standard input-output processing we are used to stream-like operations. 
```

Comment:
This section sounds a bit unstructured. Personally, I'd even consider skipping it and just referencing the relevant Javadocs, or keeping it shorter.

Reply:
Sure, removed it and added nio javadocs link. It's not enough for clear comprehensive explanation and making it shorter won't help either.